Working/pacman/Model.java

```diff
@@ -264,36 +264,37 @@ private void movePacman() {
                 pacman.addScore(1);
             }
 
-            if (req_dx != 0 || req_dy != 0) {
-                if (!((req_dx == -1 && req_dy == 0 && (ch & 1) != 0)
-                        || (req_dx == 1 && req_dy == 0 && (ch & 4) != 0)
-                        || (req_dx == 0 && req_dy == -1 && (ch & 2) != 0)
-                        || (req_dx == 0 && req_dy == 1 && (ch & 8) != 0))) {
-                    pacman.setDX(req_dx);
-                    pacman.setDY(req_dy);
+            if (invoker.isCommandSet()) {
+                if(!((invoker.getCommandName().equals("LeftMove") && (ch & 1) != 0)
```

Comment:
Galetu tiesiog but right left up down be move

Reply:
Klases pervadinti?